resources/msi/source.wxs.erb

```diff
@@ -12,23 +12,57 @@
           Version="$(var.VersionNumber)" Manufacturer="!(loc.ManufacturerName)" UpgradeCode="$(var.UpgradeCode)">
 
     <!--
-      Define the minimum supported installer version (2.0).
-      The install should be done for the whole machine not just the current user
+      Minimum installer version (5.0) - Window 7/Server 2008 R2 and above.
+      The install scope is per machine, not the current user
     -->
-    <Package InstallerVersion="200" InstallPrivileges="elevated" Compressed="yes" InstallScope="perMachine" />
+    <Package InstallerVersion="500" InstallPrivileges="elevated"
+             Compressed="yes" InstallScope="perMachine" />
 
     <Media Id="1" Cabinet="Project.cab" EmbedCab="yes" CompressionLevel="high" />
 
-    <!-- Major upgrade -->
-    <Upgrade Id="$(var.UpgradeCode)">
-      <UpgradeVersion OnlyDetect="yes" Minimum="$(var.VersionNumber)" IncludeMinimum="no" Property="NEWERVERSIONDETECTED" />
-      <UpgradeVersion Minimum="0.0.0.0" IncludeMinimum="yes" Maximum="$(var.VersionNumber)" IncludeMaximum="yes" Property="OLDERVERSIONBEINGUPGRADED" />
-    </Upgrade>
+    <!-- Launch condition to check for minimum OS: Windows 7/Server 2008R2. -->
+    <Condition Message="!(loc.MinimumOSVersionMessage)">
+      <![CDATA[Installed OR VersionNT >= 601]]>
```

Comment:
Hmm, hang on - actually I don't know for sure that the CA will work as is for Windows prior to Window 7/Win 2008 R2.  Since it is targeting .Net 2.0 now, it may require additional work to either try and re-target for .Net 1.0, or have multiple versions, or build a bootstrapper for older versions. It's easy enough to change the minimum OS though either ways, so it things do work and we want to support earlier platforms, I would propose we leave in the minimum OS, but relax it to a version earlier to pick up Windows 2008 and Vista if we want to go that far back :)